src/api/libraryManageApi.ts

```diff
@@ -53,15 +54,24 @@ const useDeleteBookMutation = () => {
 const useGetBorrowInfoListQuery = ({ page, size = 10, status, search }: BorrowInfoListSearch) => {
   const fetcher = () =>
     axios.get('/manage/borrow-infos', { params: { page, size, status, search } }).then(({ data }) => {
-      const content = data.content.map((borrowInfo: BorrowInfo) => ({
-        borrowInfoId: borrowInfo.borrowInfoId,
-        status: borrowInfo.status === '대출대기중' ? '대출 신청' : '반납 신청',
-        requestDatetime: DateTime.fromISO(borrowInfo?.requestDatetime || '').toFormat('yyyy.MM.dd'),
-        bookTitle: borrowInfo.bookTitle,
-        author: borrowInfo.author,
-        bookQuantity: '3/3', // `${borrowInfo.currentQuantity}/${borrowInfo.totalQuantity}`,
-        borrowerRealName: borrowInfo.borrowerRealName,
-      }));
+      const content = data.content.map((borrowInfo: BorrowInfo) => {
+        let statusFront = '';
+
+        if (borrowInfo.status === '대출대기중') {
+          statusFront = '대출 신청';
+        } else if (borrowInfo.status === '반납대기중') {
+          statusFront = '반납 신청';
+        }
+        return {
+          borrowInfoId: borrowInfo.borrowInfoId,
+          status: statusFront,
```

Comment:
```suggestion
        const borrowStatus = {
          대출대기중: '대출 신청',
          반납대기중: '반납 신청'
        }

        return {
          borrowInfoId: borrowInfo.borrowInfoId,
          status: borrowStatus[borrowInfo.status],
```

이런 식으로는 어떤가요? (깃헙에서 바로 작성해서 동작하는 코드인지는 한 번 확인해주세요!)

Reply:
오홍!!! 좋습니다!!!

Reply:
넘넘 좋습니다!! 수정했습니다!